test/integration/sequelize.test.js

```diff
@@ -425,46 +425,28 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
       });
     });
 
-    it('throw an exception if `values` and `options.replacements` are both passed', function() {
-      var self = this;
-      expect(function() {
-        return self.sequelize.query({ query: 'select ? as foo, ? as bar', values: [1, 2] }, { raw: true, replacements: [1, 2] });
-      }).to.throw(Error, 'Both `sql.values` and `options.replacements` cannot be set at the same time');
+    it('reject promise with error if `values` and `options.replacements` are both passed', function() {
+      this.sequelize.query({ query: 'select ? as foo, ? as bar', values: [1, 2] }, { raw: true, replacements: [1, 2] }).should.be.rejectedWith(Error, 'Both `sql.values` and `options.replacements` cannot be set at the same time');
```

Comment:
And I think you still need to wrap `sequelize.query` to `expect`, `should` assertion are working without that?